docs/parameterization-documentation.md

```diff
@@ -0,0 +1,98 @@
+# Parameterized Assets Guide
+
+This guide will help you understand how to set up and use HTTP Data assets in our system.
+
+## What are Parameterized Assets?
+
+These are HTTP Data assets where you can change certain settings, or "parameters".
+
+## Settings You Can Change
+
+when you're choosing a HTTP Data asset in the user interface (UI), there are options you can turn on or off to let you
+change certain parts of the HTTP request. These are called "overridable" fields because you can change, or "override,"
+the default settings. Here's what each option does:
+
+### Method overridability:
+if you're using the asset and you've enabled "method overridability," you must also specify a "Custom HTTP Method."
+
+### Path overridability:
+If turned on, you can add more to the "path" of the request. The path is the part of the URL
```

Comment:
Not 100% correct. You can add a path to the base url+path chosen by the provider. You cannot overwrite the whole path after URL, just add a path to the end of the base URL